tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -173,7 +208,11 @@ private void produceRecord() {
     private void withStableConsumerGroup(Runnable body) {
         Consumer<byte[], byte[]> consumer = createConsumer(new Properties());
         try {
-            TestUtils.subscribeAndWaitForRecords(TOPIC, consumer, DEFAULT_MAX_WAIT_MS);
+            consumer.subscribe(Collections.singletonList(TOPIC));
+            ConsumerRecords<byte[], byte[]> records = consumer.poll(Duration.ofMillis(DEFAULT_MAX_WAIT_MS));
+            if (records.isEmpty()) {
```

Comment:
Could we rewrite it by `assertNotEquals(0, records.count())`?

Reply:
Hi @chia7712, thanks for the review. I have addressed all following comments.